package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "single-spa-aurelia",
+  "version": "0.0.0",
+  "description": "???",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/single-spa/single-spa-aurelia.git"
+  },
+  "license": "MIT",
+  "scripts": {
+    "prebuild": "rm -rf dist",
+    "build": "rollup -c rollup.config.js",
+    "format": "prettier './**/*' --write"
+  },
+  "dependencies": {
+    "aurelia-bootstrapper": "^2.3.3"
```

Comment:
I'd rather not add a dependency until we're actually using it. Additionally, I have aversion to using dependencies like this for the following reasons:

1. UMD bundles generally are rather hard to get to work as global vars when they have external dependencies.
2. The user cannot choose their own version of aurelia-bootstrapper, but are stuck with the one we choose. This is important for users on varying (especially older) versions of frameworks.
3. It's much harder to be resilient to breaking changes in frameworks when there's a hard dependency on a version, which is a major source of pain for the single-spa core team (re: Angular).
4. Generally what we prefer doing is have the user provide the necessary parts of the framework dynamically (dependency injection, but not the angular way), which simplifies the maintenance of the package:

```js
const lifecycles = singleSpaReact({
  React,
  ReactDOM,
})

const lifecycles = singleSpaVue({
  Vue,
})
```

^ we violated this for Angular because Angular cannot work without the assumption of angular compiler. However, we should try starting single-spa-aurelia off on the right foot.